frontend/class-frontend.php

```diff
@@ -615,7 +615,8 @@ public function force_wp_title() {
 	 */
 	public function debug_mark( $echo = true ) {
 		$marker = sprintf(
-			'<!-- This site is optimized with the ' . $this->head_product_name() . '%1$s - https://yoast.com/wordpress/plugins/seo/ -->',
+			'<!-- This site is optimized with the %1$s %2$s - https://yoast.com/wordpress/plugins/seo/ -->',
```

Comment:
There are now two spaces before the version number.

Reply:
Good catch. Fixed & as I was checking the `head_product_name()` method now anyway, also improved the other comment marker. Feel free to squash the commits on merge.